test/httpClient/package.json

```diff
@@ -3,6 +3,7 @@
   "packageManager": "yarn@3.6.3",
   "private": true,
   "dependencies": {
+    "jest": "^29.7.0",
```

Comment:
This should use the same jest version as the other packages, to avoid pulling multiple versions in dependencies.